intellij-plugin/src/main/resources/messages/ApolloBundle.properties

```diff
@@ -32,11 +32,16 @@ ApolloMigrationRefactoringProcessor.codeReferences=Items to be migrated
 ApolloV2ToV3MigrationProcessor.title=Migrate to Apollo Kotlin 3
 ApolloV2ToV3MigrationProcessor.noUsage=No Apollo Android 2 references found in the project
 
+settings.codegen.title=Code Generation
 settings.codegen.automaticCodegenTriggering.text=Automatic code generation
 settings.codegen.automaticCodegenTriggering.comment=Enabling this setting will run Gradle in continuous mode, similarly to \
   <code>./gradlew generateApolloSources --continuous</code>.<br>\
   This watches changes to your GraphQL files and re-generates models when they change.
-settings.codegen.title=Code Generation
+
+settings.graphqlPlugin.title=GraphQL Plugin
+settings.graphqlPlugin.contributeConfigurationToGraphqlPlugin.text=Contribute configuration to the GraphQL plugin
+settings.graphqlPlugin.contributeConfigurationToGraphqlPlugin.comment=Allows the GraphQL plugin to be aware of the location of your GraphQL files. This invokes Gradle once per Apollo module when the project is opened, which may take a while if you have many modules.
```

Comment:
Could this be done during Gradle sync?

Reply:
> it executes Gradle once per Apollo module

This is the part that gets me confused. Gradle Sync already retrieves a bunch of the Gradle model like task names, Kotlin source sets, test names and what not. Getting the value of `schemaFile` and `serviceName` doesn't sound like it should make your CPU burn?

Reply:
> doesn't sound like it should make your CPU burn?

No I guess not 😅. It does take ~1s / module though.